core/src/main/java/com/novoda/noplayer/mediaplayer/AndroidMediaPlayerFacade.java

```diff
@@ -211,6 +213,11 @@ void stop() {
         return trackSelector.getAudioTracks(mediaPlayer);
     }
 
+    public List<PlayerSubtitleTrack> getSubtitleTracks() {
+        NoPlayerLog.w("Subtitles not implemented for Android Media Player");
```

Comment:
This is not an exception, the code can continue and doesn't need to recover. There are no available subtitle tracks in the MediaPlayer implementation. I might agree with @Dorvaryn rising the level of the log to an error though.